# PlaySong.srv
string song_name
---
bool success
